package.json

```diff
@@ -41,7 +41,7 @@
   "devDependencies": {
     "chai": "^4.1.0",
     "mocha": "^6.2.2",
-    "semver": "^6.3.0",
+    "semver": "^7.5.3",
```

Comment:
Semver is used in index.js when the module is loaded. Should this be in `dependencies` instead of `devDependencies`?

Reply:
The latest release to npm was v4.2.2 in 2017, with a non-dev dependency on semver ^5.4.1
After that, there were multiple changes that never got released, including [this commit](https://github.com/Jeff-Lewis/cls-hooked/commit/0ff594bf6b2edd6fb046b10b67363c3213e4726c) which moved
semver from dependencies to devDependencies. But that was never released and the latest code still uses semver during module load. Looks strange.